libs/stats/odc/stats/model.py

```diff
@@ -41,17 +42,20 @@ def __init__(self, start: Union[str, datetime],
         DateTimeRange(datetime(2019, 3, 1), '3M')
 
         """
+        anchor = None
```

Comment:
I don't understand this anchor thing.

anchor is only meaningful when you don't have an exact period, where you say something like "starting from the first period for which data exists give me 3month periods starting in March". Then anchor would be "March".

this class encodes exact period, so no anchor. anchor is a wild-card for generating periods not a property of a period.

Reply:
I had used the DateTimeRange class to accommodate anchor.  I've now separated 'period' from 'tempoal_range'.  Note that temporal_range is equivalent of the period in the initial implementation.